### ISSUE TITLE
TPU support?

### ISSUE THREAD

**Issue by OhadRubin** [luyug/GC-DPR#5]
Will this work on a TPU?
If so, can I train it on a colab notebook?

**Comment by luyug** [comment]
Yes, via [Tevatron](https://github.com/texttron/tevatron) toolkit
